data-share/data-share-service/src/main/java/io/mosip/datashare/util/PolicyUtil.java

```diff
@@ -108,4 +119,42 @@ public void emptyPartnerPolicyCache() {
 
 	}
 
+	/**
+	 * Provides static data share policy for sharing the data.
+	 * @param policyId Policy Id from request
+	 * @param subscriberId Subscriber Id from request
+	 * @return the DataShareDto object
+	 */
+	public DataShareDto getStaticDataSharePolicy(String policyId, String subscriberId) {

```

Comment:
Are we using the cache or each time getting from config?

Reply:
We are not causing cache and it reading each time. Vishwa suggested to keep like this.

Reply:
okay